### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by daheller99**
Issue: [OTTDEV-1037](https://vimean.atlassian.net/browse/OTTDEV-1037)

Moved required arguments to top of Create Customer section.

[OTTDEV-1037]: https://vimean.atlassian.net/browse/OTTDEV-1037?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -534,48 +526,67 @@ <h5 class="head-5 text--white margin-bottom-medium">Create a customer</h5>
...
         <span class="is-block text--transparent text-3">string</span>
         <span class="text--transparent text-3">optional</span>
       </td>
+      <td>The customer&rsquo;s full name (first and last).</td>
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">password</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional, default is &ldquo;standard&rdquo;</span>
```
Comment (jnoconor) [inline build/index.html]:
```suggestion
        <span class="text--transparent text-3">optional</span>
```

File: build/index.html [hunk 2]
```diff
@@ -534,48 +526,67 @@ <h5 class="head-5 text--white margin-bottom-medium">Create a customer</h5>
...
+      </td>
       <td>
-        This parameter is used for letting our system know when a user should lose access. This can not be updated so we do not recommend using this for managing renewals as it could lead to service interruptions for your customers.
-        <br><br>
-        Valid formats are a number followed by a unit of time. For example, <code>3-months</code>, <code>1-years</code>, <code>5-weeks</code>, <code>7-days</code>.
+        This must be at least eight characters long. Omitting this field will prompt the Customer to set their password the next time they sign in.
```
Comment (jnoconor) [inline build/index.html]: I think we typically have customer as lowercase:
```suggestion
        This must be at least eight characters long. Omitting this field will prompt the customer to set their password the next time they sign in.
```

File: build/index.html [hunk 3]
```diff
@@ -821,6 +832,18 @@ <h5 class="head-5 text--white margin-bottom-medium">Update a customer</h5>
...
+      <td class="nowrap">
+        <strong class="is-block text--navy">plan</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional, default is &ldquo;standard&rdquo;</span>
+      </td>
+      <td>
+        The customer&rsquo;s plan determines content accessibility. Values can be one of the following strings: <code>free</code>, <code>standard</code>.
+        <br><br>
+        This should be set depending on your product type. Use <code>free</code> if you are using subscription that is free with registration. In any other case, you should use <code>standard</code>.
+      </td>
```
Comment (jnoconor) [inline build/index.html]: This looks like a copy of the plan row, when it should be the password row.

**Review by jnoconor — CHANGES_REQUESTED** [review]

File: build/index.html (hunk 3 above)
Comment (daheller99) [reply build/index.html]: Updated this in newest commit. Thanks!

**Review by jnoconor — APPROVED** [review]
Nice 👏

File: build/index.html [hunk 4]
```diff
@@ -821,6 +832,16 @@ <h5 class="head-5 text--white margin-bottom-medium">Update a customer</h5>
...
       <td>The customer&rsquo;s full name (first and last).</td>
     </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">password</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+        This must be at least eight characters long. Omitting this field will prompt the customer to set their password the next time they sign in.
```
Comment (hwangjf) [inline build/index.html]: `Omitting this field will prompt the customer to set their password the next time they sign in.` This isn't the current behavior. The password is optional and won't be modified unless it is passed in, but does not cause the customer to be prompted to set their password on next login. Is that the functionality we want?

Comment (hwangjf) [reply build/index.html]: Another thing I noticed is that the Customer update docs are incorrect in that you cannot modify the plan via that endpoint. I'm guessing it was incorrectly copied over from the Customer create portion of the docs.

**Review by hwangjf — APPROVED** [review]

**Pull request merged**